src/PhpDoc/PHPUnit/MockObjectTypeNodeResolverExtension.php

```diff
@@ -37,8 +37,8 @@ public function resolve(TypeNode $typeNode, NameScope $nameScope): ?Type
 		}
 
 		static $mockClassNames = [
-			'PHPUnit_Framework_MockObject_MockObject' => true,
```

Comment:
Why did you remove support for `PHPUnit_Framework_MockObject_MockObject`?

Reply:
it was a mistake, sorry ; it's fixed.